app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -53,9 +56,26 @@ data class RecentPatient(
               WHERE facilityUuid = :facilityUuid
               GROUP BY patientUuid
           ) PD ON P.uuid = PD.patientUuid
+          LEFT JOIN (
+            SELECT MAX(updatedAt) latestUpdatedAt, T.*
+              FROM Appointment T
+              WHERE facilityUuid = :facilityUuid
+              GROUP BY patientUuid
+          ) AP ON P.uuid = AP.patientUuid
+          LEFT JOIN (
+            SELECT MAX(updatedAt) latestUpdatedAt, T.*
+              FROM Communication T
+              GROUP BY appointmentUuid
+          ) COMM ON AP.uuid = COMM.appointmentUuid
+          LEFT JOIN (
+            SELECT MAX(updatedAt) latestUpdatedAt, T.*
+              FROM MedicalHistory T
+              GROUP BY patientUuid
+          ) MH ON P.uuid = MH.patientUuid
         WHERE (
           BP.facilityUuid = :facilityUuid OR
-          PD.facilityUuid = :facilityUuid
```

Comment:
Do we even need this `WHERE` clause ?
We are already checking `facilituUuid` in individual `LEFT JOIN` queries.
Please add a test to verify this.

Reply:
Added test, seems like this WHERE clause is required. Need to investigate why.